api.bs

```diff
@@ -142,8 +167,147 @@ The aggregation service:
 
 # API Details # {#api}
 
+Open questions:
+*   Filter/query language
+*   Reports are sent to aggregation system directly, or via conversion site? Or
+    option of either? => via conversion site
+*   Epochs
+
 TODO
 
+## ListAggregationSystems API ## {#list-aggregation-systems-api}
+
+navigator.privateAttribution.listAggregationSystems()
+
+<xmp class=idl>
+dictionary PrivateAttributionAggregationSystem {
+  required DOMString id;
+};
```

Comment:
TODO: Work out whether a PrivateAttributionAggregationSystem (wow, right name, hard to type) needs any additional attributes.

Reply:
Is it possible that we can omit the `PrivateAttribution` prefix from these definitions? I'm not sure where all they might flow to and whether there is namespacing other than from the prefix.

I ask because I think `PrivateAttributionImpressionOptions` needs to be further refined as something like `PrivateAttributionImpressionOptionsLevel1` (to specify the bag of attributes for the level 1 API), which is really quite a mouthful.